public/components/CampaignPerformanceBreakdown/CampaignPerformanceBreakdown.js

```diff
@@ -14,32 +14,66 @@ class CampaignPerformanceBreakdownTable extends React.Component {
     const totalTimeSpentOnPage = analytics.map( ([key, values]) => values.timeSpentOnPage).reduce(sum, 0)
     const dataUnavailable = 'Unavailable';
 
-    return (
-      <table className="pure-table">
-        <thead>
-        <tr>
-          <th>{this.props.breakdownLabel}</th>
-          <th>Unique Users</th>
-          <th>Total Page Views</th>
-          <th>Average Time on Page (minutes)</th>
-        </tr>
-        </thead>
-
-        <tbody>
-        {analytics.map(([breakdownKey, values]) => {
-          return(
-            <tr key={breakdownKey}>
-              <td>{breakdownKey}</td>
-              <td>{values.uniques ? `${values.uniques.toLocaleString()} (${percentageOfTotal(values.uniques, totalUniques)}\%)` : dataUnavailable}</td>
-              <td>{values.pageviews ? `${values.pageviews.toLocaleString()} (${percentageOfTotal(values.pageviews, totalPageviews)}\%)` : dataUnavailable}</td>
-              <td>{values.timeSpentOnPage ? `${formatToMinutes(values.timeSpentOnPage)} (${percentageOfTotal(values.timeSpentOnPage, totalTimeSpentOnPage)}\%)` : dataUnavailable}</td>
-            </tr>
-          );
-
-        })}
-        </tbody>
-      </table>
-    );
+    if(this.props.breakdownLabel=='Path'){
```

Comment:
What other values can this have?

Reply:
ie when wouldn't you show these columns?

Reply:
Answer: `Country` and `Device`

See: https://github.com/guardian/campaign-central/blob/master/public/components/CampaignPerformanceBreakdown/CampaignPerformanceBreakdown.js#L73-L82

Reply:
Makes sense.  Thanks!